minesweeper.py

```diff
@@ -8,14 +8,19 @@
 from tkinter import PhotoImage
 from enum import Enum
 
+try:
+    image_dir = pathlib.Path(sys._MEIPASS)
+except AttributeError:
+    image_dir = pathlib.Path(__file__).parent
+
 # Recursion limit is increased to prevent Recursion error from
 # auto-opening open_squares in open_squares ()
 sys.setrecursionlimit(2000)
-where_this_file_is = pathlib.Path(__file__).parent
+image_dir = pathlib.Path(__file__).parent
```

Comment:
Delete this line of code. `image_dir` is already set above when this runs, and now it gets overwritten with the wrong path.